### DISCUSSION TITLE
Custom Table with both checkbox and radio button

### DISCUSSION THREAD

**Discussion by ghost** [quasarframework/quasar#9064] [CLI - PWA mode]
Hello friends 

I want to create a table with contain both radio button and checkbox. I did not find anything related to that can someone please help 

Thanks

**Comment by ontwikkelfabriek** [comment]
You can use for example the `body-cell-actions` slot.

Here's a demo:
https://codepen.io/ontwikkelfabriek/pen/LYxvxKB
